src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -32,24 +32,32 @@ using namespace ov;
 using namespace ov::frontend::tensorflow;
 
 namespace {
-std::vector<std::string> get_unconverted_types_from_model(const std::shared_ptr<Model>& model) {
-    std::vector<std::string> unconverted_ops_types;
+
+void get_unsupported_operations_and_failures(const std::shared_ptr<Model>& model,
+                                             std::vector<std::string>& unsupported_operations,
+                                             std::unordered_map<std::string, std::string>& failures) {
+    unsupported_operations.clear();
+    failures.clear();
     for (const auto& node : model->get_ordered_ops()) {
         if (const auto& fw_node = ov::as_type_ptr<FrameworkNode>(node)) {
             auto op_type = fw_node->get_decoder()->get_op_type();
-            unconverted_ops_types.push_back(op_type);
+            auto fw_node_attrs = fw_node->get_attrs();
+            if (fw_node_attrs.find(FrameworkNode::failed_conversion_key) != fw_node_attrs.end()) {
+                // that means the translator is found but the conversion is failed
+                failures[op_type] = fw_node_attrs.at(FrameworkNode::failed_conversion_key);
+            } else if (std::find(unsupported_operations.begin(), unsupported_operations.end(), op_type) ==
+                       unsupported_operations.end()) {
+                // found new unsupported operation
+                unsupported_operations.push_back(op_type);
```

Comment:
Can we use a set for collecting unsupported operations instead of doing not efficient linear search in `std::vector`? After collecting them in the set, you will move them to `unsupported_operations` in the end. Or just consider replacing the type of the original parameter to set instead of vector.

Reply:
Now some tests rely on the order of unsupported operations. I would rather to fix in the scope of 107500 ticket.